client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -43,7 +43,11 @@ const DnsList = React.createClass( {
 	addDns: function( record ) {
 		addDnsAction( this.props.selectedDomainName, record, ( error ) => {
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'The DNS record could not be restored.' ) );
+			} else {
+				this.props.successNotice( this.translate( 'The DNS record has been restored.' ), {
+					duration: 5000
```

Comment:
I see in https://github.com/Automattic/wp-calypso/pull/1916/files#diff-1087f0d6b5f0568a7e0bc11905363abcR152 that you can also have error notice in `props`. I'd really like we are consistent within the same module.